core/src/main/resources/FrankFrameworkConfigurationContext.xml

```diff
@@ -149,7 +149,19 @@
 	/>
 
 	<bean
-		name="proto-alarm,proto-clearing,proto-trigger"
+		name="proto-alarm"
+		class="nl.nn.adapterframework.monitoring.Alarm"
+		autowire="byName"
+		scope="prototype"
+	/>
+	<bean
+		name="proto-clearing"
+		class="nl.nn.adapterframework.monitoring.Clearing"
+		autowire="byName"
+		scope="prototype"
+	/>
```

Comment:
Nu we het interface ITrigger hebben, zou het niet nodig moeten zijn om hier aparte entries voor Alarm en Clearing te hebben